Signal/src/ViewControllers/InviteFlow.swift

```diff
@@ -88,17 +92,16 @@ class InviteFlow: NSObject, MFMessageComposeViewControllerDelegate, MFMailCompos
     func contactsPicker(_: ContactsPicker, didSelectMultipleContacts contacts: [Contact]) {
         Logger.debug("\(TAG) didSelectContacts:\(contacts)")
 
-        self.presentingViewController.dismiss(animated: true)
-
         guard let inviteChannel = channel else {
             Logger.error("\(TAG) unexpected nil channel after returning from contact picker.")
+            self.presentingViewController.dismiss(animated: true)
             return
         }
 
         switch inviteChannel {
         case .message:
             let phoneNumbers: [String] = contacts.map { $0.userTextPhoneNumbers.first }.filter { $0 != nil }.map { $0! }
-            dismissAndSendSMSTo(phoneNumbers: phoneNumbers)
```

Comment:
I pulled the dismissal logic from `dismissAndSendSMSTo` into `sendSMSTo` so that we coordinate the dismissal and presentation.

Reply:
This turned out to be a mistake. Reverted in https://github.com/signalapp/Signal-iOS/commit/5361cd3f25f151e1e057ab323374d882c8935f3b